docker/buildmaster/Dockerfile

```diff
@@ -1,3 +1,29 @@
+FROM alpinelinux/build-base:latest-x86_64 AS apk-builder
+
+RUN abuild-apk add -u \
+	gcc \
+	git \
+	linux-headers \
+	lua5.3-dev \
+	lua5.3-lzlib \
+	make \
+	musl-dev \
+	openssl-dev \
+	openssl-libs-static \
+	zlib-dev \
+	zlib-static \
+	zstd-dev \
+	zstd-static
+
+ARG APK_TOOLS_COMMIT=54caa31be633efc5f655700b77af290124f71689
+
+RUN git clone https://gitlab.alpinelinux.org/alpine/apk-tools.git /tmp/apk-tools
+WORKDIR /tmp/apk-tools
+RUN git fetch --depth=1 origin $APK_TOOLS_COMMIT
+RUN git reset --hard $APK_TOOLS_COMMIT
```

Comment:
Is this step required? I thought a single checkout already does the magic?

Reply:
Yes, that commit is not available in any branch.